core/modules/replica/SqlApp.h

```diff
@@ -59,9 +59,18 @@ class SqlApp : public Application {
 
     SqlApp(int argc, char* argv[]);
 
+    std::string _command;
     std::string _mysqlUser;
     std::string _mysqlPassword;
     std::string _query;
+    std::string _database;
+    std::string _table;
+    std::string _engine;
+    std::string _schemaFile;
+    std::string _partitionByColumn;     
```

Comment:
In principle they all are well documented in the `cc` file of the application class. Each of these members maps to the command-line parameters, options or flags. For string types I chose not to document them in a header since the names are supposed to be "self explanatory".